### PR TITLE
Issue/175/coveralls

### PR DESCRIPTION
**Pull request by m-aguena**
Increase coverage as much as possible.

### CODE REVIEW

**Comment by combet** [comment]
Hi - coveralls is failing with GA because of a recent update they made. This fix I used in branch issue/353/... did the trick: https://github.com/LSSTDESC/CLMM/pull/355/commits/ea6f0ea92a733da4ae76204da6071e4d7a92d786

File: clmm/support/mock_data.py [hunk 1]
```diff
@@ -134,20 +136,20 @@ def generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, zsrc, Delta_
...
         for badid, replacement in zip(badids, replacements):
             for col in galaxy_catalog.colnames:
                 galaxy_catalog[col][badid] = replacement[col]
+        nbad, badids = _find_aphysical_galaxies(galaxy_catalog, zsrc_min)
 
     # Final check to see if there are bad galaxies left
-    nbad, _ = _find_aphysical_galaxies(galaxy_catalog, zsrc_min)
     if nbad > 1:
-        print("Not able to remove {} aphysical objects after {} iterations".format(nbad, nretry))
+        warnings.warn("X Not able to remove {} aphysical objects after {} iterations".format(nbad, nretry))
```
Comment (combet) [inline clmm/support/mock_data.py:152]: Typo in the warning message: need to remove 'X'

File: clmm/theory/cluster_toolkit.py [hunk 2]
```diff
@@ -81,8 +81,6 @@ def eval_3d_density(self, r3d, z_cl):
                 self.cdelta, Omega_m, delta=self.delta_mdef)*h**2
 
     def eval_surface_density(self, r_proj, z_cl):
-        if self.cosmo is None:
-            raise ValueError(f"Missing cosmology.")
         h = self.cosmo['h']
```
Comment (combet) [inline clmm/theory/cluster_toolkit.py:84]: Is that check not needed anymore? Or did you move it somewhere else?

File: tests/test_galaxycluster.py [hunk 3]
```diff
@@ -1,12 +1,12 @@
 """
 Tests for datatype and galaxycluster
 """
+import numpy as np
```
Comment (combet) [inline tests/test_galaxycluster.py:4]: New numpy import, but not needed anywhere in `test_galaxycluster.py` Please remove.

File: clmm/support/mock_data.py (hunk 1 above)
Comment (m-aguena) [reply clmm/support/mock_data.py:152]: Good catch. Fixed.

**Review by combet — CHANGES_REQUESTED** [review]
Whaooo, 100% coverage !! Thanks @m-aguena, that's great.

I have 2 or 3 super minor comments/corrections above. I'll approve the PR right after.

File: clmm/theory/cluster_toolkit.py (hunk 2 above)
Comment (m-aguena) [reply clmm/theory/cluster_toolkit.py:84]: This check is not really needed here, I am guessing it was a leftover from some previous version

File: tests/test_galaxycluster.py (hunk 3 above)
Comment (m-aguena) [reply tests/test_galaxycluster.py]: done

**Comment by m-aguena** [comment]
@combet all comments solved

**Review by combet — APPROVED** [review]
All good - thanks!

**Pull request merged**